MorphicServer/Endpoint.cs

```diff
@@ -98,9 +112,29 @@ public static async Task Run<T>(HttpContext context) where T: Endpoint, new()
             endpoint.Context = context;
             endpoint.Request = context.Request;
             endpoint.Response = context.Response;
+            var method = context.Request.Method;
+            var statusCode = 500;
+            var pathAttr = endpoint.GetType().GetCustomAttribute(typeof(Path)) as Path;
+            var path = context.Request.Path.ToString();
+            if (pathAttr?.Template == null)
```

Comment:
`pathAttr` should be non-null, or else the endpoint wouldn't have been matched, although I get the compiler doesn't know about that.  I'm not sure why `Path.Template` was defined as nullable because it returns a private non-nullable field.  I'd say redefine `Path.Template` to be non-nullable.

Reply:
Done.